### PR TITLE
feature/add pick-up delivery

### PR DESCRIPTION
**Pull request by kajawitek**
Hi Nerds 🤓

I added pick-up delivery method to the app.

AC:

- A reward should have a new delivery method: pick-up
- When an employee chooses pick-up he should receive an email with pick-up instructions

### CODE REVIEW

File: app/controllers/admins/orders_controller.rb [hunk 1]
```diff
@@ -22,5 +22,18 @@ def deliver
         redirect_to admins_orders_url, notice: "Order wasn't delivered"
       end
     end
+
+    def prepare_to_pick_up
+      @order = Order.find(params[:id])
+      begin
+        ActiveRecord::Base.transaction do
+          EmployeeMailer.reward_pick_up_instruction_email(@order).deliver
```
Comment (piowit) [inline app/controllers/admins/orders_controller.rb:30]: mailers can be send outside transaction - also we should send email after we prepare_to_pick_up

File: app/views/admins/pages/_orders.html.erb [hunk 2]
```diff
@@ -16,15 +16,22 @@
   <tbody>
     <% orders.each do |order| %>
       <tr>
-        <td><%= order.delivery_status %></td>
-        <td><%= order.reward.delivery_method %></td>
+        <td><%= order.delivery_status.humanize %></td>
+        <td><%= Reward.delivery_methods[order.reward.delivery_method] %></td>
```
Comment (piowit) [inline app/views/admins/pages/_orders.html.erb:20]: this can be also `.humanize`

**Review by piowit — APPROVED** [review]
looks good

File: app/controllers/admins/orders_controller.rb (hunk 1 above)
Comment (kajawitek) [reply app/controllers/admins/orders_controller.rb]: done ✅

File: app/views/admins/pages/_orders.html.erb (hunk 2 above)
Comment (kajawitek) [reply app/views/admins/pages/_orders.html.erb]: done ✅

**Pull request merged**